.env.template

```diff
@@ -10,4 +10,7 @@ TS_OUTBOX=/var/tms-data/po_files/outbox
 TRANSLATION_GOOGLE_KEY=/var/tms/secrets/TranslationGoogleKey.json
 WP_IMPORT_URL=https://ctoassetsstg.wpengine.com
 WP_IMPORT_USER=your-wpengine-username
-WP_IMPORT_PASSWORD='your-wpengine-password'
\ No newline at end of file
+WP_IMPORT_PASSWORD='your-wpengine-password'
+WP_EXPORT_URL=
+WP_EXPORT_USER=
+WP_EXPORT_PASSWORD=
```

Comment:
Should we just use the same WP_URL/WP_USER/WP_PASSWORD for both import and export?